artview/components/tools.py

```diff
@@ -58,6 +60,8 @@ def reset_tools(tooldict):
     -----
     Disconnects all tools.
     '''
+    import warnings
+    warnings.warn("this function is deprecated")
     for tool in tooldict:
         if tooldict[tool] is not None:
             tooldict[tool].disconnect()
```

Comment:
Just curious: what's the reason for putting the tool disconnects in the display scripts versus in the tools.py?